### DISCUSSION TITLE
CU DU split implementation.

### DISCUSSION THREAD

**Discussion by safwan2m** [srsran/srsRAN_Project#455] [General Help] [answered]
I wanted to know if CU DU split is available in the code. If so, how can go about running them in split configuration. I see it as one of the upcoming features. But also see some of the code instantiating the CU object and DU object. Just wanted a clarification. Thanks in advance.

**Comment by dvdgrgrtt** [comment]
The CU and the DU exist as functional entities, separated to one another. However, in the current version of the code, they run in the same binary. The full split, with CU and DU running as two different binaries, is a feature we are working on.

**Reply by peihaoY** [reply]
Hi [dvdgrgrtt](https://github.com/dvdgrgrtt), could you please tell me when the separation of CU and DU will be available in the code? possibly provide a rough timeline? This would be greatly beneficial to my research. Thanks.

**Reply by ismagom** [reply, marked as answer]
We expect it for Release 24.10. Regards

**Reply by hpate063** [reply]
Quick Question: 
Is it possible to split the existing code into two binaries using hacks to have separate CU and DU binaries (even though both binaries can run the entire code)? 
If so, would anyone share any idea / hints on how this can be done?
